project/Build.scala

```diff
@@ -262,6 +262,7 @@ object BijectionBuild extends Build {
       "org.apache.hadoop" % "hadoop-core" % "1.0.4" % "provided->default"
     )
   ).dependsOn(bijectionCore % "test->test;compile->compile")
+   .dependsOn(bijectionMacros % "test->test;compile->compile")
```

Comment:
I think this is just `.dependsOn(bijectionMacros)` you don't need to depend on the test code (as we are doing in these other examples).